package-testing/dotnet-sdk-relay/EppoSDKRelay/DTO/BanditActionRequest.cs

```diff
@@ -21,6 +21,6 @@ public ContextAttributes GetSubjectContext()
     public IDictionary<string, ContextAttributes> GetActionContextDict()
     {
         return Actions.ToDictionary(action => action.ActionKey,
-                                    action => ContextAttributes.FromDict(action.ActionKey, Values.ConvertJsonValuesToPrimitives(action.AttributeDictionary)));
```

Comment:
combining the already sorted attributes causes issues because they're then separated by type by `ContextAttributes`. There's an overlap of types where non-numeric attributes can be passed to the API but they are expected to be ignored. This was part-way attempt to get around the lack of a true dynamic-typing API in the SDK. 
The function call below is available in the latest version of the SDK.